app/javascript/components/widgets/twitter/tweet_body.jsx

```diff
@@ -1,39 +1,53 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import Highlighter from 'react-highlight-words';
-import { fontSizes, colors } from '../../../lib/theme';
+import { TweetText, TweetMedia, Text } from './tweet_content';
+import { colors, fontSizes } from '../../../lib/theme';
 
-const TweetText = styled.div`
-  color: #959292;
-  font-size: ${fontSizes.large};
-  margin-bottom: 29px;
+const LinkText = styled(Text)`
+  color: ${colors.white};
+  text-decoration: underline;
+  font-size: ${fontSizes.medium};
 `;
 
-const highlighterStyle = {
-  backgroundColor: 'transparent',
-  color: `${colors.white}`,
-};
+export const BothMediaTypes = (displayText, mediaUrl, linkUrl) => (
+  <React.Fragment>
+    <div>
+      <TweetText text={displayText} />
+      <LinkText>{linkUrl}</LinkText>
+    </div>
+    <TweetMedia mediaUrl={mediaUrl} linkUrl={undefined} />
+  </React.Fragment>
+);
+
+export const OneMediaType = (displayText, mediaUrl, linkUrl) => (
+  <React.Fragment>
+    <div>
+      <TweetText text={displayText} />
+    </div>
+    <TweetMedia
+      mediaUrl={mediaUrl === null ? undefined : mediaUrl}
+      linkUrl={linkUrl === null ? undefined : linkUrl}
+    />
+  </React.Fragment>
+);
 
-const getMentionsAndTags = text => {
-  const allWords = text.split(/[\s,\n]+/);
-  return allWords.filter(word => word.includes('@') || word.includes('#'));
+const TweetBody = ({ text, mediaUrl, linkUrl }) => {
+  if (mediaUrl === null && linkUrl === null) return <TweetText text={text} />;
```

Comment:
Prefer using brackets for these blocks.

Reply:
Do you mean the conditionals? Because if so, they're against the current eslint rules

Reply:
No errors on my end. What does your editor show you?
![condition-single-lines](https://user-images.githubusercontent.com/524850/60992686-01229980-a31b-11e9-900c-795e9b858b44.png)

Reply:
I'm not getting errors but whenever I hit save, prettier removes them. I'll check my rules